### ISSUE TITLE
Show severity levels directly in the output

### ISSUE THREAD

**Issue by KabbAmine** [Kuniwak/vint#131]
Hi first of all thanks for your linter :+1: 

Without using json output, its hard to get severity level (error, warning ...), so can we have a way to output it directly?

For example ...
```
sourcePreview.vim:110:16: unexpected token: <EOL> (see ynkdir/vim-vimlparser)
```

... will become something like this:
```
sourcePreview.vim:110:16: error: unexpected token: <EOL> (see ynkdir/vim-vimlparser)
```

Its will be easier then to integrate `vint` into external tools.

**Comment by Kuniwak** [comment]
This feature released with v0.3.3.
You can do it by using the `--format` option.

```shell
vint --format '{file_path}:{line_number}:{column_number}: {severity}: {description} (see {reference})'
```

 See: https://github.com/Kuniwak/vint/wiki/About---format-option

**Comment by KabbAmine** [comment]
Just awesome :+1: Thank you.

I close the issue.
